### DISCUSSION TITLE
google fonts

### DISCUSSION THREAD

**Discussion by HessamCheraghi** [rastikerdar/vazir-font#239] [Q&A] [answered]
برنامه ای دارید که این فونت رو به google fonts هم اضافه کنید؟

**Comment by rastikerdar** [comment, marked as answer]
فرستاده شده برای بررسی نهایی. احتمالا به زودی اونجا منتشر میشه.
https://twitter.com/rastikerdar/status/1501614095803498496

**Comment by ShahriarKh** [comment]
https://fonts.google.com/specimen/Vazirmatn
